### PR TITLE
Change default batch size to 1000

### PR DESCRIPTION
**Pull request by hovaesco**
Addresses: https://github.com/starburstdata/dbt-trino/issues/16

### CODE REVIEW

File: dbt/include/trino/macros/adapters.sql [hunk 1]
```diff
@@ -151,6 +151,11 @@
 {% endmacro %}
 
 
+{% macro trino__get_batch_size() %}
```
Comment (findinpath) [inline dbt/include/trino/macros/adapters.sql:154]: I recommend adding a documentation for this macro and its purpose in `dbt seed` command.

Please also point out in the documentation that the user can "override" this default behaviour by adding `default__get_batch_size()` macro definition on his project.
Or if you think that this is way too verbose in the code, you could add something similar to the `README.md` file in a separate section.

**Review by findinpath — APPROVED** [review]

File: dbt/include/trino/macros/adapters.sql (hunk 1 above)
Comment (hovaesco) [reply dbt/include/trino/macros/adapters.sql:154]: Done

File: README.md [hunk 2]
```diff
@@ -177,6 +177,19 @@ Check the Trino connector documentation for more information.
 }}
 ```
 
+#### Seeds
+
+Seeds are CSV files in your dbt project (typically in your data directory), that dbt can load into your data warehouse using the dbt seed command.
+
+For dbt-trino batch_size is defined in macro `trino__get_batch_size()` and default value is `1000`.
+In order to override default value define local macro like:
```
Comment (findinpath) [inline README.md:185]: In order to override default value define within your project a macro like the following

Comment (hovaesco) [reply README.md]: Addressed.

**Pull request merged**